src/server.h

```diff
@@ -361,6 +361,9 @@ typedef enum {
 /* Synchronous read timeout - slave side */
 #define CONFIG_REPL_SYNCIO_TIMEOUT 5
 
+/* The default block number of trimming replication backlog per call. */
+#define TRIM_REPL_BUF_BLOCKS_PER 64
```

Comment:
not sure i understand this one, maybe the name / doc should be improved.
```suggestion
/* The default number of replication backlog blocks to trim per call. */
#define REPL_BACKLOG_TRIM_BLOCKS_PER_CALL 64
```